wifiphisher/common/deauth.py

```diff
@@ -59,11 +59,16 @@ def _craft_and_add_packet(self, sender, receiver):
         :rtype: None
         """
 
-        packet = (dot11.RadioTap() / dot11.Dot11(type=0, subtype=12, \
-                    addr1=receiver, addr2=sender, addr3=sender) \
+        deauth_packet = (dot11.RadioTap() / dot11.Dot11(type=0, subtype=12, \
+                    addr1=receiver, addr2=sender, addr3=self._ap_bssid) \
                   / dot11.Dot11Deauth())
 
-        self._deauthentication_packets.append(packet)
+        disassoc_packet = (dot11.RadioTap() / dot11.Dot11(type=0, subtype=10, \
```

Comment:
Can you also document this change? In the method's docstring it says "Craft a deauthentication packet and add it to the list of deauthentication packets".

The module's docstring seems to need change as well.